src/charactersheet/viewmodels/character/traits/index.html

```diff
@@ -62,7 +62,7 @@
                     <tr>
                       <td width="10%" style="text-align:center">
                         <span style="opacity: .9; color: #2c3e50;"
-                          class="glyphicon glyphicon-time" aria-hidden="true"></span>
+                          class="fa fa-clock-o" aria-hidden="true"></span>
```

Comment:
I assumed all the replacement fontawesome icons are equivalent to the glyph ones, or as close as possible, right?

Reply:
Yeah. They are as close as I can get them. The only difference between the time icon and the clock icon is where the hands are.